### PR TITLE
[Merged by Bors] - Fix the tensorrt and model_compiler bugs for Adlik-0.4.0

### LINKED ISSUE #580: tensorrt runtime load error
image：[tensorrt runtime ](https://github.com/Adlik/Adlik#:~:text=registry.cn-beijing.aliyuncs.com/adlik/serving-tensorrt%3Av0.4.0_trt7.2.1.6_cuda11.0)
load tensorrt model，error as : 
2022-06-15 10:32:54: I adlik_serving/runtime/tensorrt/model/trt_model.cc:129] After CreateTrtModel, status: instance group yolov5m_trt_7.2.1_cuda_11 of model yolov5m_trt_7.2.1_cuda_11 must be KIND_GPU and must specify at least on GPU id

### PR DESCRIPTION
**Pull request by nengyinyibeiwu**
Closes #580.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,20 +80,18 @@ Docker pull command:
 
 The compiler docker images can be used in CPU and GPU. In the CPU, you can  compile the model from source type to TensorFlow
 model, OpenVino model and TensorFlow Lite model. And in the CPU, you can compile the model from source type to TensorFlow
-model, and TensorRT model. The names and labels of compiler mirrors are as follows, and the first half of label
+model, and TensorRT model. The name and label of compiler mirror is shown below, and the first half of label
```
Comment (shiyingjin) [inline README.md:83]: "is shown" should be "are shown"

File: docker-images/dockerfiles/serving-tensorrt.dockerfile [hunk 2]
```diff
@@ -66,7 +68,8 @@ RUN env PYTHON_BIN_PATH=/usr/bin/python3 \
          -c opt \
          --action_env=LIBRARY_PATH=/usr/local/cuda-${CUDA_VERSION}/lib64/stubs \
          --incompatible_use_specific_tool_files=false \
-         --copt="-Ithird_party/cuda/include"
+         --copt="-Ithird_party/cuda/include" \
+         --jobs=3
```
Comment (shiyingjin) [inline docker-images/dockerfiles/serving-tensorrt.dockerfile:72]: Why "--jobs=3"?

**Review by shiyingjin — CHANGES_REQUESTED** [review]
In  README.md  “git clone https://github.com/ZTE/Adlik.git”  should be " https://github.com/Adlik/Adlik.git"

File: docker-images/dockerfiles/model-compiler.dockerfile [hunk 3]
```diff
@@ -33,18 +33,19 @@ ARG TENSORRT_VERSION
...
-    apt-get autoremove --purge -y wget && \
-    apt-get clean && \
-    find /var/lib/apt/lists -delete
-
-RUN . /etc/os-release && \
-    echo "deb https://developer.download.nvidia.com/compute/cuda/repos/$ID$(echo $VERSION_ID | tr -d .)/x86_64 /\n\
-    deb https://developer.download.nvidia.com/compute/machine-learning/repos/$ID$(echo $VERSION_ID | tr -d .)/x86_64 /" >> /etc/apt/sources.list
+	apt-get update && \
+	apt-get install --no-install-recommends -y \
+		gnupg curl software-properties-common && \
```
Comment (NaNAGISaSA) [inline docker-images/dockerfiles/model-compiler.dockerfile:41]: Better to merge these two lines:
`
	apt-get install --no-install-recommends -y \
		gnupg curl software-properties-common && \
`
into one line.

File: docker-images/dockerfiles/serving-tensorrt.dockerfile [hunk 4]
```diff
@@ -14,15 +14,16 @@ COPY script/run_server.sh /script/run_server.sh
 
 RUN . /etc/os-release && \
     apt-get update && \
-    apt-get install --no-install-recommends -y wget ca-certificates && \
-    wget "https://developer.download.nvidia.com/compute/cuda/repos/$ID$(echo $VERSION_ID | tr -d .)/x86_64/7fa2af80.pub" -O /etc/apt/trusted.gpg.d/cuda.asc && \
+    apt-get install --no-install-recommends -y \
```
Comment (NaNAGISaSA) [inline docker-images/dockerfiles/serving-tensorrt.dockerfile:17]: Better to merge these two lines:
`
	apt-get install --no-install-recommends -y \
		gnupg curl software-properties-common && \
`
into one line.

**Review by NaNAGISaSA — CHANGES_REQUESTED** [review]

**Comment by NaNAGISaSA** [comment]
The pull request title should be modified too, describing what is really done in this PR.

File: README.md [hunk 5]
```diff
@@ -80,20 +80,18 @@ Docker pull command:
...
+registry.cn-beijing.aliyuncs.com/adlik/model-compiler:v0.4.0_trt7.1.3.4_cuda11.0
 
 #### Using model compiler image compile model
 
 1. Run the image.
 
    ```shell script
    docker run -it --rm -v source_model:/mnt/model
-   registry.cn-beijing.aliyuncs.com/adlik/model-compiler:v0.3.0_trt7.2.1.6_cuda11.0 bash
+   registry.cn-beijing.aliyuncs.com/adlik/model-compiler:v0.4.0_trt7.1.3.4_cuda11.0 bash
```
Comment (shiyingjin) [inline README.md:94]: Why tensorrt downgrade to 7.1.3.4?

File: README.md [hunk 6]
```diff
@@ -80,20 +80,18 @@ Docker pull command:
...
 The compiler docker images can be used in CPU and GPU. In the CPU, you can  compile the model from source type to TensorFlow
 model, OpenVino model and TensorFlow Lite model. And in the CPU, you can compile the model from source type to TensorFlow
-model, and TensorRT model. The names and labels of compiler mirrors are as follows, and the first half of label
+model, and TensorRT model. The name and label of compiler mirror are shown below, and the first half of label
 represents the version of TensorRT, the latter part of label represents the version of CUDA:
 
-registry.cn-beijing.aliyuncs.com/adlik/model-compiler:v0.3.0_trt7.2.1.6_cuda11.0
-
-registry.cn-beijing.aliyuncs.com/adlik/model-compiler:v0.3.0_trt7.2.1.6_cuda10.2
+registry.cn-beijing.aliyuncs.com/adlik/model-compiler:v0.4.0_trt7.1.3.4_cuda11.0
```
Comment (shiyingjin) [inline README.md:86]: Why tensorrt downgrade to 7.1.3.4?

**Review by shiyingjin — CHANGES_REQUESTED** [review]

File: docker-images/build.py [hunk 7]
```diff
@@ -6,8 +6,8 @@
 from os import path
 
 _UBUNTU_VERSION = 'bionic'
-# _TENSORRT_VERSION = '7.2.1.6'
-_TENSORRT_VERSION = '7.1.3.4'
+_TENSORRT_VERSION = '7.2.1.6'
+# _TENSORRT_VERSION = '7.1.3.4'
```
Comment (shiyingjin) [inline docker-images/build.py:10]: Remove the commented code

**Review by shiyingjin — CHANGES_REQUESTED** [review]

**Review by shiyingjin — APPROVED** [review]

File: docker-images/build.py [hunk 8]
```diff
@@ -6,8 +6,7 @@
 from os import path
 
 _UBUNTU_VERSION = 'bionic'
-# _TENSORRT_VERSION = '7.2.1.6'
-_TENSORRT_VERSION = '7.1.3.4'
+_TENSORRT_VERSION = '7.2.1.6'
 _CUDA_VERSION = '11.0'
 # _CUDA_VERSION = '10.2'
```
Comment (shiyingjin) [inline docker-images/build.py:11]: Remove the commented code  "# _CUDA_VERSION = '10.2'"

**Review by shiyingjin — CHANGES_REQUESTED** [review]

**Review by shiyingjin — APPROVED** [review]

**Review by NaNAGISaSA — APPROVED** [review]

**Pull request closed without merging**